### PR TITLE
Bug 1908367: pipeline triggers scripts

### PR DESCRIPTION
**Pull request by makambalaji**
Jira Id: https://issues.redhat.com/browse/ODC-5195
Bugzilla Id; https://bugzilla.redhat.com/show_bug.cgi?id=1908367

Please Note: These scripts are not integrated with CI
If need to verify, execute the below command "yarn run test-cypress-dev console" on fronted
On cypress UI, please select the appropriate file

Test Results:
No. of smoke test cases in this pr: 8 [Execution Time: sec]
No. of regression test cases in this or: 3 [Execution Time: ]
No. of test cases in this pr: 11 [Execution Time: ]

Defect Raised :

### CODE REVIEW

**Comment by spadgett** [comment]
/override ci/prow/ceph-storage-plugin

The ceph-storage-plugin job should only have run on PRs with changes to `frontend/packages/ceph-storage-plugin`. It failed due to an unrelated CI config change. Overriding status on this PR.

File: frontend/packages/dev-console/integration-tests/support/step-definitions/pipelines/pipelines-triggers.ts [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  'option {string} is not available in kebab menu for {string}',
+  (option: string, pipelineName: string) => {
+    pipelinesPage.selectKebabMenu(pipelineName);
+    cy.byTestActionID(option).should('not.be.visible');
+  },
+);
+
+Given('Trigger is added to the pipeline {string} present in pipeline details page', (a: string) => {
+  cy.log(a);
+  // Manual step
```
Comment (karthikjeeyar) [inline frontend/packages/dev-console/integration-tests/support/step-definitions/pipelines/pipelines-triggers.ts:244]: @makambalaji  Is this just a placeholder and will there be a separate PR to add content?

Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/step-definitions/pipelines/pipelines-triggers.ts:244]: This is manual step, To avoid warnings in feature file, I added its step in step definition and commented as manual step @karthikjeeyar
